### PR TITLE
Support bindable default value for inputs

### PR DESCRIPTION
**Pull request by Janpot**
split out default value in a separate prop and make it bindable to e.g. page parameters

Fixes https://github.com/mui/mui-toolpad/issues/414

### CODE REVIEW

**Comment by Janpot** [comment]
@apedroferreira Trying out applying the methods from https://playwright.dev/docs/test-pom here

File: test/integration/runtime-basic/basicDom.json [hunk 1]
```diff
@@ -0,0 +1,176 @@
+{
```
Comment (apedroferreira) [inline test/integration/runtime-basic/basicDom.json:1]: in my PR i gave the file a more explicit name like `twoTextFields`, maybe it helps understand better the contents of the DOM without having to look at the JSON content itself.

File: test/integration/runtime-basic/index.spec.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import { ToolpadHome } from '../../models/ToolpadHome';
+import { test, expect } from '../../playwright/test';
+import basicDom from './basicDom.json';
+
+const PAGE_ID = 'vlpwdwr';
```
Comment (apedroferreira) [inline test/integration/runtime-basic/index.spec.ts:5]: would it be worth getting this directly from the DOM json?

**Review by apedroferreira — APPROVED** [review]
seems good! just added some comments about testing
lmk when you get the runtime env variable running in CI

**Comment by apedroferreira** [comment]
One more comment about UX: is it intentional that the prop control is completely disable on the right? I thought we wanted to have two-way binding, but not sure how easy or doable it is for now.

File: packages/toolpad-app/src/config.ts [hunk 3]
```diff
@@ -65,11 +63,11 @@ function getBrowsersideRuntimeConfig(): RuntimeConfig {
   return maybeRuntimeConfig;
 }
 
-export const runtimeConfig: RuntimeConfig =
+const runtimeConfig: RuntimeConfig =
   typeof window === 'undefined'
     ? {
         // Define runtime config here
-        foo: process.env.TOOLPAD_FOO,
+        integrationTest: !!process.env.TOOLPAD_INTEGRATION_TEST,
```
Comment (apedroferreira) [inline packages/toolpad-app/src/config.ts:70]: was something like this what i was missing for CI to work?

Comment (Janpot) [reply packages/toolpad-app/src/config.ts:70]: We were using built-time configuration for this, and we build the docker image for production, so it was set to false

**Pull request merged**